### DISCUSSION TITLE
cannot invite a user to join organization

### DISCUSSION THREAD

**Discussion by ghost** [community/community#36379] [Projects and Issues]
I have canceled a invition unexpected to click 'opt-out'.
<img width="576" alt="image" src="https://user-images.githubusercontent.com/115976659/196122870-3e3198a6-c230-48bf-9a48-938dbf7862c2.png">

And the owner of the organization can't invite me again.
<img width="923" alt="image" src="https://user-images.githubusercontent.com/115976659/196123001-d54f40a2-d6a9-4154-9e9b-e1f5cf7687c2.png">

Help me please...

**Comment by ossanna16** [comment]
Hey there @arfon and thank you for asking a good question! I'm glad you found our community 🙂
